configs/opencv-ovms/cmd_client/res/object_detection/configuration.yaml

```diff
@@ -1,3 +1,6 @@
 OvmsClient:
   PipelineScript: run_object_detection.sh
-  PipelineInputArgs: "" # space delimited like we run the script in command and take those input arguments  
\ No newline at end of file
+  PipelineInputArgs: "" # space delimited like we run the script in command and take those input arguments
+  EnvironmentVariableFiles:
+    - test1.env
```

Comment:
why are there 2 env files for object detection?

Reply:
probably only 1 - i am just showing an example that we can take multiple env files if needed.

Reply:
gst profile has examples of two env files so multiple env files support is needed.